package.json

```diff
@@ -43,6 +43,11 @@
   },
   "devDependencies": {
     "@fast-check/jest": "^1.1.0",
+    "@peculiar/asn1-pkcs8": "^2.3.0",
+    "@peculiar/asn1-schema": "^2.3.0",
+    "@peculiar/asn1-x509": "^2.3.0",
+    "@peculiar/webcrypto": "^1.4.0",
+    "@peculiar/x509": "^1.8.3",
```

Comment:
FYI, you need to include `"pvtsutils": "1.3.2",` with this. Otherwise the crypto stuff gets a broken dependency and doesn't work. 

You may be getting errors when using the certificates otherwise.